WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -1512,8 +1515,8 @@ private void setCurrentTag(final ReaderTag tag) {
                 mTagPreviewHistory.push(tag.getTagSlug());
                 break;
             case BLOG_PREVIEW:
-                break;
```

Comment:
BTW, I'd personally add a separate comment to each of the no-op switch cases instead of "sharing" the comment between the cases (the `BLOG_PREVIEW` now actually needs a comment for why the fall-through to an unrelated switch case) but, this is not something I'd block the PR on.